pkg/generate/app/cmd/newapp_test.go

```diff
@@ -137,6 +176,7 @@ func TestValidate(t *testing.T) {
 			}
 		}
 	}
+	envFileOperation("/tmp/envfile123", "delete")
```

Comment:
Sure once I have a variable for this, I will use TempFile.Name(), thats in plan already. I didn't understand the defer part, its already outside of loop and this way even if the test case fails, we will delete that file off and function is having a defer when it is deleting it.